robot_ws/src/voice_interaction_robot/scripts/audio_input.py

```diff
@@ -37,9 +37,9 @@
 DEFAULT_ASSETS_DIR = rospkg.RosPack().get_path('voice_interaction_robot') + "/assets"
 DEFAULT_ASSETS_EXT = ".wav"
 
-audio_input_publisher = rospy.Publisher("/audio_input", AudioData, queue_size=5)
-audio_output_publisher = rospy.Publisher("/audio_output", AudioData, queue_size=5)
-wake_publisher = rospy.Publisher("/wake_word", String, queue_size=5)
+audio_input_publisher = rospy.Publisher("/audio_input", AudioData, 5)
+audio_output_publisher = rospy.Publisher("/audio_output", AudioData, 5)
+wake_publisher = rospy.Publisher("/wake_word", String, 5)
```

Comment:
Other than the rospy->rclpy issue, should we specify a suitable QoS profile? For this case I think QoSReliabilityPolicy makes sense: https://github.com/ros2/rclpy/blob/dashing/rclpy/rclpy/qos.py#L274

Same suggestion applies to all other publishers/subscribers we create too.

Reply:
I'll do this in a future PR and spend some time learning QoS profiles and trying them first.